packages/react-storage/src/components/StorageBrowser/views/LocationActionView/CreateFolderView/__tests__/CreateFolderView.spec.tsx

```diff
@@ -156,7 +156,7 @@ describe('CreateFolderView', () => {
       data: {
         isActionStartDisabled: true,
         folderNameId,
-        messageContent: 'Success!',
+        message: 'Success!',
```

Comment:
This is the wrong shape for `message`, we must be covering up the TS error with a cast somewhere above maybe?